internal/provider/resource_monitor.go

```diff
@@ -401,6 +401,40 @@ var monitorSchema = map[string]*schema.Schema{
 		Optional:    true,
 		Computed:    true,
 	},
+	"environment_variables": {
+		Description: "For Playwright monitors, the environment variables that can be used in the scenario.",
```

Comment:
💄 Feel free to include an example here as well, it's great to see an example straight in the docs. OK if omitted for a reason 👍 

```suggestion
		Description: "For Playwright monitors, the environment variables that can be used in the scenario. Example: `{ "PASSWORD" = "passw0rd" }`.",
```

Reply:
Done. The example was omitted because I copied this from the API docs, where the example wasn't included in all places. This is now also fixed.